style.css

```diff
@@ -14,22 +14,18 @@ main {
 }
 
 main div {
-    color: #3f8abf;
-    font-weight: 500;
-    font-size: 1.25em;
-    text-align: center;
-    line-height: 80px;
-    width: 80px;
-    height: 80px;
+    width: 55px;
+    height: 55px;
     display: inline-block;
-    background-color: white;
+    text-align: center;
     border-radius: 50%;
-    margin: 0.5em;
+    line-height: 55px;
+    margin: 5px;
+    text-shadow: 0px 1px 2px rgba(0, 0, 0, 0.25);
```

Comment:
This is debatable but generally I try to sort things by alphabetical order.
You could also order by 'priority' but thats a little more subjective.

Reply:
Not my code, copied from TH directly but good to know.

Reply:
Changed to alphabetical order.